platformio.ini

```diff
@@ -57,99 +54,30 @@ lib_deps =
           https://github.com/anklimov/ModbusMaster
           https://github.com/anklimov/DMXSerial
           https://github.com/anklimov/Ethernet
-          https://github.com/PaulStoffregen/SPI.git
-          https://github.com/knolleary/pubsubclient.git
-          https://github.com/anklimov/Artnet.git
-          FastLED
-        ;  161
-        ;  322
-
-[env:controllino]
-platform = atmelavr
-board = megaatmega2560
-framework = arduino
-build_flags = !echo -n "-DPIO_SRC_REV="$(git rev-parse --short HEAD)
-build_flags = -D CONTROLLINO -D CUSTOM_FIRMWARE_MAC=de:ad:be:ef:fe:07
-lib_deps =
```

Comment:
need to return this targets back (controllino, *-5500)

Reply:
OK.